ChangeLog.d/x509write_csr_heap_alloc.txt

```diff
@@ -0,0 +1,4 @@
+Changes
```

Comment:
Sorry, I'm having second thoughts: I think it would be better to make that "Bugfix", for consistency with the ChangeLog entry in 3488, and because a "Changes" entry in an LTS branch is quite unusual.

(Arguably the entry in development could be "Bugfix" too, but now that the PR is merge I don't think it's worth worrying about it.)

Reply:
Sure, I've updated entry to reflect that.